src/eventHandlers/checkSuite/index.ts

```diff
@@ -128,15 +127,18 @@ const tryMerge = async (
   }
 };
 
-export const checkSuiteHandle = async (octokit: GitHub): Promise<void> => {
+export const checkSuiteHandle = async (
+  octokit: GitHub,
+  GITHUB_LOGIN: string,
```

Comment:
It's not a constant here, it's a parameter.

```suggestion
  gitHubLogin: string,
```

Reply:
fixed with https://github.com/ridedott/dependabot-auto-merge-action/pull/233/commits/8fa422481dda410cd7bc52b6c04ead5cbcab9c54